### ISSUE TITLE
Windows only: Cannot read files larger than 4GB

### ISSUE THREAD

**Issue by torognes** [torognes/vsearch#291]
If your try to read from a FASTA file larger than 4GB on Windows, VSEARCH will terminate with a fatal error and the message "Unable to seek in input file (filename)". This is due to the following code on line 177-183 in fastx.cc that tries to find the file size.

```
  if (! h->is_pipe)
    {
      if (fseek(h->fp, 0, SEEK_END))
        fatal("Unable to seek in input file (%s)", filename);
      h->file_size = ftell(h->fp);
      rewind(h->fp);
    }
```

Apparently the fseek function does not work on large files on Windows.

There are some suggestions for resolving this problem here:

https://stackoverflow.com/questions/9026896/get-large-file-size-in-c

**Comment by torognes** [comment]
Fixed in latest commit, but not released yet.

**Comment by torognes** [comment]
Fixed in VSEARCH 2.7.1 just released.
